mne/viz/ica.py

```diff
@@ -59,6 +60,11 @@ def plot_ica_sources(ica, inst, picks=None, start=None,
         plotter. For evoked, this parameter has no effect. Defaults to False.
     show_first_samp : bool
         If True, show time axis relative to the ``raw.first_samp``.
+    show_real_time : bool | str
+        If True or a `datetime format code <https://docs.python.org/3/library
+        /datetime.html#strftime-and-strptime-format-codes>`_, the labels for
+        x-axis ticks will be the real time derived from the Info['meas_date'].
+        Default format code if True will be ``H:M:S``.
```

Comment:
To escape `%` when you plan on using `%` formatting later (this is what `docdict` / `docs.py` does under the hood) you double `%`:
```
>>> '%(test)s %%H:%%M%%S' % dict(test='whatever')
'whatever %H:%M%S'
```
so:
```suggestion
        Default format code if True will be ``'%%H:%%M:%%S'``.
```

Reply:
> 
> 
> your doc CIs are green. Do you have this pb locally?

Yes, its locally (on Windows). Maybe I redo the mnedev-environment and see if the pb persists.

Reply:
Thank you!